plugins/mac-capture/mac-screen-capture.m

```diff
@@ -1009,7 +1049,19 @@ static bool content_settings_changed(void *data, obs_properties_t *props,
 		props, "show_hidden_windows",
 		obs_module_text("WindowUtils.ShowHidden"));
 
+	obs_properties_add_bool(props, "show_cursor",
+				obs_module_text("DisplayCapture.ShowCursor"));
+
+	obs_property_t *hide_obs = obs_properties_add_bool(
+		props, "hide_obs", obs_module_text("DisplayCapture.HideOBS"));
+
 	if (sc) {
+		obs_property_set_modified_callback2(
+			capture_type, content_settings_changed, sc);
+
+		obs_property_set_modified_callback2(
+			hide_obs, content_settings_changed, sc);
```

Comment:
I want to apologize for not realizing earlier, but looking at this again I’m not sure it needs a modified_callback in the first place. As far as I can tell (and I haven’t tested this myself yet), the only things that need these callbacks are properties which have a direct effect on other properties, e.g. the capture type changing which other comboboxes are shown, or the "Show hidden windows" toggle changing the list of available windows.

Reply:
Given that the checkbox only affects display capture, the other dropdowns are unaffected by this change, correct.